LandMark/App/AppCoordinator.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AppCoordinator.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 30/10/24.
+//
+
+import UIKit
+
+protocol Coordinator {
+    var children: [Coordinator] { get set }
+    var navigationController: UINavigationController { get set }
+    func start()
+}
+
+class AppCoordinator: Coordinator {
+    var children: [Coordinator] = .init()
+    
+    var navigationController: UINavigationController
```

Comment:
Baseado no comentário acima:
```suggestion
    private(set) var children: [Coordinator] = .init()
    
    private(set) var navigationController: UINavigationController
```